examples/bri-3/prisma/prisma.mapper.ts

```diff
@@ -1,37 +1,56 @@
 import { Injectable } from '@nestjs/common';
-import { BpiSubject as BpiSubjectPrismaModel, BpiSubjectRole as BpiSubjectRolePrismaModel } from '@prisma/client';
+import {
+  BpiSubject as BpiSubjectPrismaModel,
+  BpiSubjectRole as BpiSubjectRolePrismaModel,
+  PublicKey as PublicKeyPrismaModel,
+} from '@prisma/client';
 import { BpiSubject } from '../src/bri/identity/bpiSubjects/models/bpiSubject';
 import { BpiSubjectRole } from '../src/bri/identity/bpiSubjects/models/bpiSubjectRole';
+import { PublicKey } from '../src/bri/identity/bpiSubjects/models/publicKey';
 
 // We do mapping from prisma models to domain objects using simple Object.assign
 // since automapper is not happy working with types, which is how Prisma generates database entities.
 // For these mappings to work, the convention is that the domain objects have the same properties as the
-// prisma models. In case there is a need to do something custom during mapping, it can be implemented 
-// in the appropriate method below. 
+// prisma models. In case there is a need to do something custom during mapping, it can be implemented
+// in the appropriate method below.
 
 interface IConstructor<T> {
-    new (...args: any[]): T;
+  new (...args: any[]): T;
 }
 
 @Injectable()
 export class PrismaMapper {
-    public mapBpiSubjectPrismaModelToDomainObject(source: BpiSubjectPrismaModel): BpiSubject {
-        const target = this.activator(BpiSubject);
+  public mapBpiSubjectPrismaModelToDomainObject(
```

Comment:
This conflicts with @ognjenkurtic PR #780 @biscuitdey ... it might be better to adopt that approach here already or wait for the other PR to be merged.

Reply:
Mine is merged, you can rebase your PR on top of changes from main. @biscuitdey

Reply:
Done.